api-server/common/models/user.js

```diff
@@ -331,9 +335,10 @@ export default function(User) {
   });
 
   User.doesExist = function doesExist(username, email) {
-    if (!username && (!email || !isEmail(email))) {
+    if (!username.toLowerCase() && (!email || !isEmail(email))) {
```

Comment:
```suggestion
    if (!username && (!email || !isEmail(email))) {
```

You may not have the object itself first.

Reply:
@raisedadead trying to commit your suggestion but seems I don't have permission to push to the repo. Can you commit them and @ojeytonwilliams 's suggestions?

Reply:
@nikrb have you tried pushing a new commit onto the branch?  Sometimes the Github web interface just doesn't work.

Reply:
@ojeytonwilliams I tried a local push originally but it said I didn't have permission. Is that what you mean?

Reply:
Yep, you should have been able to push your local changes. I'll take a look when I can and try and see what's going on.